src/config.rs

```diff
@@ -2160,6 +2160,7 @@ mod readpool_tests {
 pub struct BackupConfig {
     pub num_threads: usize,
     pub batch_size: usize,
+    pub region_max_size: ReadableSize,
```

Comment:
this seems irrelevant to `coprocessor.region-max-size` besides the init value

perhaps better name it something like `sst_max_size` / `backup_file_max_size` / `sst_max_kv_size` / ...